assign_rights/views.py

```diff
@@ -28,24 +28,74 @@ def get_context_data(self, **kwargs):
         return context
 
 
-def RightsShellListView(ListView):
+class RightsShellListView(LoginRequiredMixin, ListView):
     '''browse and search rights shells'''
-    pass
+    queryset = RightsShell.objects.all()
```

Comment:
You can also do 
```suggestion
    model = RightsShell
```
(see GroupingListView below) - whatever we choose we should just be consistent.